setup.cfg

```diff
@@ -0,0 +1,56 @@
+[metadata]
```

Comment:
Might be wrong here, but I'm fairly certain we don't need this file. We're currently using `setup.py` instead of a `.cfg`. [relevant SO question?](https://stackoverflow.com/questions/39484863/whats-the-difference-between-setup-py-and-setup-cfg-in-python-projects)

Reply:
This was again mentioned in the now missing comment. I used `setup.cfg` because it was the only way I could figure out to save the prefs we need for the various linters. From what research I did online, Python packaging seems to be in a bit of flux right now and `setup.cfg` vs `setup.py` is up in the air.

There _are_ dependencies, and possibly other issues, like version numbers, in both files, which that needs to be sorted out. There are also settings in `setup.cfg` for `mypy` and `pydantic` even though I haven't managed to add those into the code yet.

This is a decision that needs to be made soon, though.

Reply:
resolving this for now but made #24 to track this